Lib/test/test_nntplib.py

```diff
@@ -343,8 +344,9 @@ class NetworkedNNTP_SSLTests(NetworkedNNTPTests):
     # 400 connections per day are accepted from each IP address."
 
     NNTP_HOST = 'nntp.aioe.org'
-    GROUP_NAME = 'comp.lang.python'
-    GROUP_PAT = 'comp.lang.*'
+    GROUP_NAME = 'aioe.test'
```

Comment:
It's probably a good idea to add a comment to this section, mentioning that aioe.test is one of the official groups on this server used for testing: https://news.aioe.org/manual/aioe-hierarchy/

Reply:
If you add a comment, add "bpo-42794: " which always helps for future readers ;-)